src/lib/actions.ts

```diff
@@ -232,7 +232,7 @@ export async function updateStropheColor(studyId: string, selectedStrophes: Stro
   }
 }
 
-export async function updateStropheState(studyId: string, stropheId: number, newState: boolean) {
+export async function updateStropheState(studyId: string, strophe: StropheData, newState: boolean) {
```

Comment:
Why do we need to pass the strophe data into here? Wouldn't strophe id be sufficient?